### DISCUSSION TITLE
Difference between mkdocstrings.github.io and mkdocstrings.github.io/mkdocstrings

### DISCUSSION THREAD

**Discussion by connesy** [mkdocstrings/mkdocstrings#714] [Documentation]
When looking at the documentation, there is a mismatch between mkdocstrings.github.io and mkdocstrings.github.io/mkdocstrings, even though it seems they should be identical(?)

**Comment by pawamoy** [comment]
Oh, good catch, there should only be https://mkdocstrings.github.io/, the other docs were probably deployed by mistake, and I'll try to remove them :slightly_smiling_face:

**Reply by pawamoy** [reply]
Fixed! Thanks again :smile:

**Reply by pawamoy** [reply]
And I made sure it won't happen again: https://github.com/mkdocstrings/mkdocstrings/commit/6561ad5e1afa272c7cc0d53e33150fb75fe72e3e

**Reply by connesy** [reply]
Sure, happy to help 🙂
